Microsoft.Azure.Devices.Edge.sln

```diff
@@ -217,6 +217,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EdgeHubTriggerCSharp", "edg
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NumberLogger", "test\modules\NumberLogger\NumberLogger.csproj", "{B0151FFD-85C4-42B9-81CC-9193BADFE9F8}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "jsoncanonicalizer", "edge-util\src\jsoncanonicalizer\jsoncanonicalizer.csproj", "{1915CE7E-E949-4633-9862-D92BD5E873B5}"
```

Comment:
Can we keep this copy and let tools referencing source code at edgeutil? This way, we don't need to keep the same reference code in two places.

Reply:
I didn't mean the solution file.... 

I meant we are keeping the same source code under both edgeutil and tools\manifestsigner folder. 
Instead we can just keep the one under edgeutil; and have tools\manifestsigner reference it.